great_expectations/expectations/expectation.py

```diff
@@ -174,12 +175,16 @@ class MetaExpectation(ABCMeta):
     attribute, or, if that is not set, by snake-casing the name of the class.
     """
 
+    default_kwarg_values: Dict[str, object] = {}
```

Comment:
```suggestion
    default_kwarg_values: Dict[str, Union[str, bool, None]] = {}
```

???

Reply:
Could also be `int`, `float`, or anything really. Think of all of the possibilities for Expectation parameter defaults.